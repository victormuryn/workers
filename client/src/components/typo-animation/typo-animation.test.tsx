import React from 'react';
import renderer from 'react-test-renderer';
import TypoAnimation from './typo-animation';

describe(`TypoAnimation tests`, () => {
  it(`TypoAnimation renders corrects`, () => {
    const tree = renderer.create(<TypoAnimation
      phrases={[`test`, `second-test`]}
    />).toJSON();

    expect(tree).toMatchSnapshot();
  });

  it(`TypoAnimation with full data renders corrects`, () => {
    const tree = renderer.create(<TypoAnimation
      phrases={[`test`, `second-test`]}
      elementsClass="test-class"
      deleteSpeed={3000}
      writeSpeed={2000}
      delay={1000}
    />).toJSON();

    expect(tree).toMatchSnapshot();
  });
});
