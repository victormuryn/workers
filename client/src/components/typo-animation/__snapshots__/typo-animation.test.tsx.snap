// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TypoAnimation tests TypoAnimation renders corrects 1`] = `
<span>
  test
</span>
`;

exports[`TypoAnimation tests TypoAnimation with full data renders corrects 1`] = `
<span
  className="test-class"
>
  test
</span>
`;
